Enterprise-CMCS/eAPD
# Tbolt/4602 Update help text on cost allocation and other funding page for MMIS

Resolves #4602 

### Description
Text updates on MMIS page for cost allocation and other funding

### Chromatic Link
N/A

### Significant changes or possible side effects
None

### Automated test cases written
No new test cases written. Existing tests updated to work with text changes.

### Steps to manually verify this change

1. Login and view HITECH APD and verify existing help text not impacted (covered by existing test)
2. Login and view MMIS APD and verify help text is according to specification

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
